bin/matrix-appservice-wechaty.ts

```diff
@@ -12,6 +12,13 @@ async function main () {
     log.level(process.env.LOG_LEVEL as any)
   }
 
+  if (process.env.SIMULATED) {
+    // XXX This is aim to use the chrome referred by /usr/bin/chromium-browser as dependency of wechaty.node_modules.puppeteer module.
+    Object.defineProperty(process, 'arch', {
```

Comment:
I don't think it's a good idea to modify `process.arch` in our code.

It looks like a workaround and we should find a solution instead.